service/src/main/kotlin/fi/espoo/evaka/attendance/RealtimeStaffAttendanceController.kt

```diff
@@ -173,12 +176,13 @@ class RealtimeStaffAttendanceController(
                             occupancyCoefficientSeven
                         )
                     }
+                    Pair(staffAttendanceIds, externalStaffAttendanceIds)
```

Comment:
Nit: Why not return a list here, since the `Pair` is deconstructed only to later be put in a list?